### PR TITLE
Remove related charms from entity data request.

### PR DESCRIPTION
**Pull request by kadams54**
Fixes https://github.com/juju/juju-gui/issues/1479.

### CODE REVIEW

**Comment by kadams54** [comment]
## QA

Pull up several entity details pages and make sure nothing breaks.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4694/
Test PASSed.

File: jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js [hunk 1]
```diff
@@ -373,10 +373,6 @@ var module = module;
           location: extraInfo['bzr-url']
         }
       };
-      if (meta['charm-related']) {
-        this._lowerCaseKeys(meta['charm-related'], meta['charm-related']);
-        processed.relatedCharms = meta['charm-related'];
```
Comment (hatched) [inline jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js]: Did you forget to commit a file? In charm.js there are a number of methods which deal with the `relatedCharms` property.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4697/
Test PASSed.

File: jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js (hunk 1 above)
Comment (kadams54) [inline jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js]: Nope, just completely missed that as I was grepping for 'charm-related'. Thanks!

**Comment by huwshimi** [comment]
Thanks @kadams54 :+1:

**Comment by frankban** [comment]
:+1:

**Comment by hatched** [comment]
:ok: Thanks @kadams54 This really speeds up charm details page loads.
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**